tests/manual/testmargindrift.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta http-equiv="content-type" content="text/html; charset=UTF-8">
+		<meta name="viewport" content="initial-scale=1">
+	</head>
+		<body>
```

Comment:
Please do not use tabs to indent, normal 4 spaces for each indentation level.

Reply:
Thanks; this should be fixed now.